behat/tests/features-v2/reporting/sections/visits-care/visits-care.feature

```diff
@@ -46,21 +34,16 @@ Feature: Visits and Care
     And I view the report overview page
     Then I should see "visits-care" as "not started"
     When I view and start the visits and care report section
-    Then I choose yes and save on the live with the client section
-    Then I should be on the client receive paid care page
-    When I choose yes and all care is paid for by someone else and then save on the receive paid care section
-    Then I should be on the who is doing the caring page
+    Then I confirm I live with the client
```

Comment:
As we're not asserting on anything here, just doing scenario setup, this should probably be an `And`